Source/Core/VideoBackends/Vulkan/VKGfx.h

```diff
@@ -23,19 +19,16 @@ class VKFramebuffer;
 class VKPipeline;
 class VKTexture;
 
-class Renderer : public ::Renderer
+class VKGfx : public ::AbstractGfx
 {
 public:
-  Renderer(std::unique_ptr<SwapChain> swap_chain, float backbuffer_scale);
-  ~Renderer() override;
+  VKGfx(std::unique_ptr<SwapChain> swap_chain, float backbuffer_scale);
+  ~VKGfx() override;
 
-  static Renderer* GetInstance() { return static_cast<Renderer*>(g_renderer.get()); }
+  static VKGfx* GetInstance() { return static_cast<VKGfx*>(g_gfx.get()); }
```

Comment:
Feels unnecessary... just use `g_gfx`?

Reply:
Needs to be cast to `VKGfx` for a few places.

Reply:
Ah whoops.  I don't know how I missed that.